client/packages/host/src/components/ErrorAlert.tsx

```diff
@@ -48,7 +48,6 @@ export const ErrorAlert = () => {
     }
 
     if (error === AuthError.PermissionDenied) {
-      toggleOff();
```

Comment:
Was added to prevent alert being shown twice but is making the alert show twice 😢